components/Layout.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import Navbar from './Navbar'
+import Sidenav from './Sidenav'
+import { useRouter } from 'next/router'
+import styles from '../styles/Home.module.css'
+
+const Layout = ({ children }) => {

```

Comment:
Add props typing, if you are using Visual Studio Code always make sure that prettier + eslint are installed so you can never forget them.

Reply:
Now check your other components and functions